src/AudioContext.js

```diff
@@ -59,38 +55,36 @@ class AudioContext extends events.EventEmitter {
     if (opts.numBuffers) this.format.numBuffers = opts.numBuffers
 
     this.outStream = null
-    this._encoder = pcmUtils.BufferEncoder(this.format)
+    this._encoder = utils.BufferEncoder(this.format)
```

Comment:
The prefix can be removed with named imports (see above)

We we could use `#` instead of `_` to make real private fields

Reply:
Yep, thanks. Haven't got to all of possible private fields.
Only thing to mind is that we still need "protected" fields (not private) to be reused in descendants. Just keeping `_` for now.